### ISSUE TITLE
update vulnerable dependency (request)

### ISSUE THREAD

**Issue by quinndiggity** [twilio/twilio-node#168]
When running nsp from https://nodesecurity.io against twilio-node@2.9.1 the following report is generated:

(+) 1 vulnerabilities found
┌───────────────┬────────────────────────────────────────────┐
│                            │ Regular Expression Denial of Service                         │
├───────────────┼────────────────────────────────────────────┤
│ Name                  │ hawk                                                                          │
├───────────────┼────────────────────────────────────────────┤
│ Installed              │ 2.3.1                                                                          │
├───────────────┼────────────────────────────────────────────┤
│ Vulnerable          │ < 3.1.3  || >= 4.0.0 <4.1.1                                        │
├───────────────┼────────────────────────────────────────────┤
│ Patched              │ >=3.1.3 < 4.0.0 || >=4.1.1                                        │
├───────────────┼────────────────────────────────────────────┤
│ Path                    │ twilio@2.9.1 > request@2.55.0 > hawk@2.3.1         │
├───────────────┼────────────────────────────────────────────┤
│ More Info            │ https://nodesecurity.io/advisories/77                          │
└───────────────┴────────────────────────────────────────────┘

**Comment by jaumard** [comment]
Same with snyk : 

```
 snyk test twilio
✗ Low severity vulnerability found on hawk@2.3.1
- desc: Regular Expression Denial of Service
- info: https://snyk.io/vuln/npm:hawk:20160119
- from: twilio@2.9.1 > request@2.55.0 > hawk@2.3.1
Upgrade direct dependency request@2.55.0 to request@2.59.0 (triggers upgrades to hawk@3.1.3)

✗ Medium severity vulnerability found on request@2.55.0
- desc: Remote Memory Exposure
- info: https://snyk.io/vuln/npm:request:20160119
- from: twilio@2.9.1 > request@2.55.0
Upgrade direct dependency request@2.55.0 to request@2.68.0

Tested twilio for known vulnerabilities, found 2 vulnerabilities, 2 vulnerable paths.

```

**Comment by YasharF** [comment]
Related: 
https://github.com/twilio/twilio-node/pull/167
https://github.com/twilio/twilio-node/pull/154
https://github.com/twilio/twilio-node/pull/153
https://github.com/twilio/twilio-node/issues/170

**Comment by YasharF** [comment]
Related: https://github.com/twilio/twilio-node/pull/185 for fix in 3.3.0-edge branch

**Comment by YasharF** [comment]
Fixed:
https://github.com/twilio/twilio-node/commit/95cd584a90d1e6c4b8575a8451469b329a98f7fc
